.github/workflows/cut-release-branch.yml

```diff
@@ -0,0 +1,41 @@
+# **what?**
+# Cuts a new `*.latest` branch
+# Also cleans up all files in `.changes/unreleased` and `.changes/previous verion on
```

Comment:
I don't see where this cleanup happens. Perhaps this is calling a different workflow?

Reply:
Yes, everything happens in [actions/cut-release-branch.yaml](https://github.com/dbt-labs/actions/blob/main/.github/workflows/cut-release-branch.yaml)

Reply:
I updated the comment to be more clear